src/luxonis_ml/embeddings/requirements.txt

```diff
@@ -0,0 +1,15 @@
+docker>=6.1.3
+KDEpy>=1.1.5
+kmedoids>=0.4.3
+matplotlib>=3.7.2
+numpy>=1.25.2
+onnx>=1.14.0
+# onnxruntime==1.15.1
+onnxruntime-gpu>=1.15.1
+opencv-python>=4.8.0.76
+qdrant-client>=1.4.0
+scikit-learn>=1.3.0
+scipy>=1.11.1
+torch==1.13.1+cu116
```

Comment:
I would omit PyTorch dependencies here and then make it so that if we install this part of the package, we also need the loader part.